app/components/placement/summary_component.rb

```diff
@@ -1,12 +1,18 @@
 class Placement::SummaryComponent < ApplicationComponent
   with_collection_parameter :placement
-  attr_reader :placement, :school, :provider
+  attr_reader :placement, :school, :provider, :location_coordinates
 
-  def initialize(provider:, placement:, classes: [], html_attributes: {})
+  def initialize(provider:, placement:, location_coordinates: nil, classes: [], html_attributes: {})
     super(classes:, html_attributes:)
 
     @provider = provider
     @placement = placement.decorate
     @school = @placement.school
+    @location_coordinates = location_coordinates
+  end
+
+  def distance_from_location
+    distance = @school.distance_to(@location_coordinates).round(1)
+    I18n.t("components.placement.summary_component.distance_in_miles", distance:)
```

Comment:
You should be able to shorten this line with:

```suggestion
    t(".distance_in_miles", distance:)
```